### ISSUE TITLE
New UI: Send feedback really needs to be a form.

### ISSUE THREAD

**Issue by ocdtrekkie** [sandstorm-io/sandstorm#775]
A lot of users do not have working `mailto:` due to webmail clients. On my work machine, my experience was seeing the briefest of popups on Edge which promptly disappeared. No indication of what was supposed to happen was visible.

**Comment by ocdtrekkie** [comment]
Likely equally true for "Contact support".

**Comment by kentonv** [comment]
I agree this should eventually be a form but we probably aren't going to be able to implement it for launch...
